src/karmen_backend/tests/clients/test_octoprint.py

```diff
@@ -1522,3 +1522,262 @@ def test_plugin_bad_response(self, mock_post_uri):
         )
         r = printer.set_lights()
         self.assertFalse(r)
+
+
+class KarmenSniffPrinterTest(unittest.TestCase):
```

Comment:
Jen poznámka: Hodně mockuješ interní funkce, refactoring kódu bude asi znamenat hodně práce na testech. Nicméně to je hodně dané povahou aplikace jako takové.
Připadá mi, že chybí testy na chybové stavy.
Každopádně super, že tam ty testy jsou.